141. Linked List Cycle.py

```diff
@@ -14,5 +14,6 @@ def hasCycle(self, head: Optional[ListNode]) -> bool:
             else:
                 head.val=flag
             head=head.next
+            
         return False
         
```

Comment:
Overall, great work Fasil! You found a creative way of detecting a cycle in a linked list. As a suggestion, this problem can also be solved by using the slow and fast pointer technique.